rmshit.py

```diff
@@ -104,7 +113,8 @@ def rmshit():
         absf = os.path.expanduser(f)
         if os.path.exists(absf):
             found.append(absf)
-            print("    %s" % f)
+            size = get_size(absf)
+            print("    %s (%d bytes)" % (f, size))
```

Comment:
Let's make it modern:

```suggestion
            print(f"    {f} ({size} bytes)")
```

Reply:
Sure. I've seen the formatting where you use statements with %s and implemented it similarly. I'll discover the `pathlib` and improve my suggestions.